### PR TITLE
Adds math parsing

### PR DESCRIPTION
**Pull request by Silver-Sorbet**
This adds LaTex support for the project. It supports both inline and display.

### CODE REVIEW

**Comment by K4LCIFER** [comment]
For the sake of continuity, I'll reference the issue that this pull request pertains to here: https://github.com/Nutomic/ibis/issues/44

File: Cargo.toml [hunk 1]
```diff
@@ -79,6 +79,12 @@ doku = "0.21.1"
 smart-default = "0.7.1"
 tower-layer = "0.3.3"
 
+[target.'cfg(target_arch = "x86_64")'.dependencies]
+katex = { version = "0.4", features = ["duktape"], default-features = false }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+katex = { version = "0.4", features = ["wasm-js"], default-features = false }
+
```
Comment (Nutomic) [inline Cargo.toml:87]:
```suggestion
katex = { version = "0.4", default-features = false }
```

Comment (Nutomic) [reply Cargo.toml:87]: Then at the top under `[features]`, add `"katex/duktape"` under ssr and `"katex/wasm-js"` under csr. That way the correct features are enabled for backend/frontend and its less verbose.

File: src/frontend/app.rs [hunk 2]
```diff
@@ -93,6 +93,7 @@ pub fn App() -> impl IntoView {
         <>
             <Stylesheet id="simple" href="/assets/simple.css" />
             <Stylesheet id="ibis" href="/assets/ibis.css" />
+            <Stylesheet href="https://cdn.jsdelivr.net/npm/katex@0.16.11/dist/katex.min.css"/>
```
Comment (Nutomic) [inline src/frontend/app.rs:96]: Please download this and put it in the assets folder to avoid tracking etc.

**Review by Nutomic — APPROVED** [review]

**Comment by Nutomic** [comment]
Thank you!

**Pull request merged**